tests/test_HomSap.py

```diff
@@ -31,13 +31,13 @@ def test_recombination_rates(self):
         # compare the results to the current recombination rates for each chromosome
         genetic_map = "HapMapII_GRCh37"
         species = stdpopsim.get_species("HomSap")
-        for chrom in self.genome.chromosomes:
+        for chrom in self.genome.chromosomes[2:]:
```

Comment:
What's going on here with the first two chromosomes?

Reply:
Good spot! Left over from earlier debugging.